src/System.Runtime.Extensions/src/ApiCompatBaseline.uap.txt

```diff
@@ -0,0 +1,4 @@
+MembersMustExist : Member 'System.IO.Path.EndsInDirectorySeparator(System.ReadOnlySpan<System.Char>)' does not exist in the implementation but it does exist in the contract.
```

Comment:
You do not need this. The uap build flavor uses same CoreCLR as regular netcoreapp. Once the CoreCLR version is updated for netcoreapp, the uap build flavor will work fine too.

You just need the uapaot one.

Reply:
Ok just to understand...I knew that uap is uwp(I never worked with it) so this apis will be present on uwp?

Reply:
Yes, we are keeping the netcoreapp and uap public surface in sync.

Reply:
ah perfect thank's!So next question is mandatory 😄 why we need to skip on uapaot(called .NET Native?) version?

Reply:
The changes for uapaot has integration latency: CoreCLR->CoreRT->ProjectN (closed source)->official build->CoreFX

So you have two options, either:
- Add temporary  ApiCompatBaseline for uapaot and be ready to merge your change as soon as the CoreCLR build with your change shows up in CoreFX
or 
- Wait for the above to go through and then you do not need to add the ApiCompatBaseline  for uapaot

Reply:
Ah ok .Net native runtime with new Api will be consumed for uapaot corefx build(ref) as the last

Reply:
I would do the first of the two options as it will be much faster